coalib/results/Diff.py

```diff
@@ -290,9 +297,19 @@ def range(self, filename):
         >>> print(range.start.line)
         None
 
-        :param filename: The filename to associate the SourceRange with.
+        >>> range = Diff([], filename="file").range()
```

Comment:
can we put this example first and after it a note that for backwards compatability you can still call it with a filename?